tests/parser_templates/cli/show_version/16.6.4.txt

```diff
@@ -64,19 +64,18 @@ Technology Package License Information:
 -----------------------------------------------------------------
 Technology    Technology-package           Technology-package
               Current       Type           Next reboot
-              ------------------------------------------------------------------
```

Comment:
what's the reason for the formatting change here? I am guessing copypasta issue on original commit?

Reply:
Yes, it was a copy paste problem.